startups/view.py

```diff
@@ -1,16 +1,16 @@
-from .models import Startup
-from rest_framework.response import Response
 from rest_framework import status
+from rest_framework.response import Response
+
+from .models import Startup
 from .serializers import StartupListSerializer
 
 
 def list(self, request):
-    # Example URL: /api/startups/
-    # Getting ALL startups logic
-    startups = Startup.objects.all()
-
+    startups = Startup.objects.filter(is_active=True)
```

Comment:
You don't need to manually create a 404 error response. The Django REST Framework provides out-of-the-box functions for this.
You do not need to manually handle the Investor.DoesNotExist exception. Django provides a get_object_or_404 function that automatically generates an HTTP 404 response if the object is not found.
Example:
    def list(self, request):
        investors = Investor.objects.filter(is_active=True)
        serializer = InvestorSerializer(investors, many=True)
        return Response(serializer.data, status=status.HTTP_200_OK)

Reply:
Removed extra excepts. 
The function get_object_or_404 cannot be used to retrieve a list of objects.